### DISCUSSION TITLE
{"error":{"code":"504","message":"An error occurred with your deployment"}}

### DISCUSSION THREAD

**Discussion by DamnYatin** [spencerwooo/onedrive-vercel-index#871] [Q&A]
{"error":{"code":"504","message":"An error occurred with your deployment"}}

I literally put the redis url right and checked it twice and I have deployed around 4 - 5 and tried with api.json credentials and without also but still I am facing the same problem. I even changed the credentials and tried the new one but then also the same problem.

site.json pricipal user is also correct but still I am facing the problem. 
I request you to fix this as soon as possible.
![image](https://user-images.githubusercontent.com/88388062/218240363-7cb86cce-c4d4-4af6-8b84-21a7b025670b.png)

**Comment by 6uu1** [comment]
一样的问题

**Comment by rokibhasansagar** [comment]
I'm facing the same 😢

**Comment by AbdullahKhanIE** [comment]
Im facing the same

**Comment by wdw1007** [comment]
What is the format of "REDIS_URL" in the environment variable?
Try changing "redis://default:" to "rediss://:" ?

看看环境变量里的 "REDIS_URL" 是个什么格式?
把 "redis://default:" 改成 "rediss://:" 试试看?

**Comment by H539251932** [comment]
尝试重新部署了几次，仍然是Oops，这里是 404 页面这个问题
{"error":{"code":"504","message":"An error occurred with your deployment"}}
有没有大佬救救
